AY2425S1-CS2103T-W09-3/tp
# Add filter by status feature

## Description

Closes #106 


Filters internship application by status



## Changes made

 FilterCommandParser is case insensitive. Validates status using StatusValidator. 

 

## Impact of changes
 

## Testing Steps

1. run and execute command /filter pending 
2. Verify that internship applications with matching statuses are listed
